docs/events/after-tests-read.md

```diff
@@ -0,0 +1,51 @@
+# AFTER_TESTS_READ {#after-tests-read}
+
+**sync | master | worker**
+
+The `AFTER_TESTS_READ` event is triggered after the `readTests or `run` methods of the `TestCollection` object are called. The event handler is executed synchronously. The event is also available in Testplane workers.
```

Comment:
Something went wrong with the backticks.

Also I see no point in writing about synchronous nature of it and availability in master/works, since it's already specified at the top. This applies to all files.